### ISSUE TITLE
Error in f-vector computation?

### ISSUE THREAD

**Issue by jplab** [Normaliz/Normaliz#238]
The following input code seems to produce a wrong f-vector

```amb_space 30
number_field min_poly (a^4 - 5/4*a^2 + 5/16) embedding [0.951056516295154 +/- 6.36e-16]
inhom_equations 24
 (-2*a^2 + 1) (2*a^2 - 3/2) (-2*a^2 + 1) 1 (2*a^2 - 3/2) 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 (-2*a^2 + 1) 0 0 0 0 (2*a^2 - 3/2) 0 (-2*a^2 + 1) 1 (2*a^2 - 3/2) 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 (-2*a^2 + 1) 0 0 0 0 (2*a^2 - 3/2) (-2*a^2 + 1) 0 0 1 (2*a^2 - 3/2) 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 (-2*a^2 + 1) 0 0 (2*a^2 - 3/2) 0 0 0 0 0 0 (-2*a^2 + 1) (2*a^2 - 3/2) 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 (-2*a^2 + 1) 0 0 (2*a^2 - 3/2) 0 0 0 0 0 0 0 0 (-2*a^2 + 1) (2*a^2 - 3/2) 1 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 (-2*a^2 + 1) 0 0 0 0 0 0 0 (2*a^2 - 3/2) 0 0 (-2*a^2 + 1) 0 0 1 (2*a^2 - 3/2) 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 (-2*a^2 + 1) 0 (2*a^2 - 3/2) 0 0 0 0 0 0 0 0 (-2*a^2 + 1) (2*a^2 - 3/2) 1 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 (-2*a^2 + 1) 0 0 0 0 0 (2*a^2 - 3/2) 0 0 0 0 0 (-2*a^2 + 1) 0 0 1 (2*a^2 - 3/2) 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 (-2*a^2 + 1) 0 0 0 0 0 0 (2*a^2 - 3/2) 0 0 0 0 (-2*a^2 + 1) 0 0 (2*a^2 - 3/2) 1 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 (-2*a^2 + 1) 0 0 0 0 (2*a^2 - 3/2) 0 0 0 0 (-2*a^2 + 1) 0 0 0 1 (2*a^2 - 3/2) 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 (-2*a^2 + 1) 0 0 (2*a^2 - 3/2) 0 0 0 0 0 0 (-2*a^2 + 1) 1 0 (2*a^2 - 3/2) 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 (-2*a^2 + 1) 0 0 (2*a^2 - 3/2) (-2*a^2 + 1) 0 0 1 (2*a^2 - 3/2) 0
 (4*a^3 - 3*a) (a) (-4*a^3 + 3*a) 0 (-a) 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 (4*a^3 - 3*a) 0 0 0 0 (a) 0 (-4*a^3 + 3*a) 0 (-a) 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 (4*a^3 - 3*a) 0 0 0 0 (a) (-4*a^3 + 3*a) 0 0 0 (-a) 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 (4*a^3 - 3*a) 0 0 (a) 0 0 0 0 0 0 (-4*a^3 + 3*a) (-a) 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 (4*a^3 - 3*a) 0 0 (a) 0 0 0 0 0 0 0 0 (-4*a^3 + 3*a) (-a) 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 (4*a^3 - 3*a) 0 0 0 0 0 0 0 (a) 0 0 (-4*a^3 + 3*a) 0 0 0 (-a) 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 (4*a^3 - 3*a) 0 (a) 0 0 0 0 0 0 0 0 (-4*a^3 + 3*a) (-a) 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 (4*a^3 - 3*a) 0 0 0 0 0 (a) 0 0 0 0 0 (-4*a^3 + 3*a) 0 0 0 (-a) 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 (4*a^3 - 3*a) 0 0 0 0 0 0 (a) 0 0 0 0 (-4*a^3 + 3*a) 0 0 (-a) 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 (4*a^3 - 3*a) 0 0 0 0 (a) 0 0 0 0 (-4*a^3 + 3*a) 0 0 0 0 (-a) 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 (4*a^3 - 3*a) 0 0 (a) 0 0 0 0 0 0 (-4*a^3 + 3*a) 0 0 (-a) 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 (4*a^3 - 3*a) 0 0 (a) (-4*a^3 + 3*a) 0 0 0 (-a) 0
inhom_inequalities 30
 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0 0
 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0
```
It gives the following f-vector
`[7095, 34767, 70136, 75152, 46121, 16429, 3378, 411, 30, 1]`
Whereas it should look like:
`(1, 1, 278, 2340, 6616, 8812, 6105, 2192, 375, 30, 1)`

**Comment by w-bruns** [comment]
Yes, there is a bug in Normaliz 3.7.x.

The bug has been corrected in the current master (by commit 25644eb043ea1430c3a188e28fcccc87dab2e3cf)

**Comment by w-bruns** [comment]
Bug fixed in 3.7.4.

**Comment by fingolfin** [comment]
If @jplab can confirm that this works for them in 3.7.4, then this issue can be closed, I think?

**Comment by jplab** [comment]
I'll be able to confirm this by next week (currently away from computer).

**Comment by jplab** [comment]
I confirm that this error is fixed with 3.7.4 while using the normaliz interface inside sage.
